As specified in the smartmontools PSARC case, add a warning message to the
smartctl man page to make sure that users are aware of the consequences of
using this utility.

This change is not yet appropriate for upstream.

--- smartmontools-6.5/smartctl.8.in.orig	2017-01-23 09:41:39.205216399 +0000
+++ smartmontools-6.5/smartctl.8.in	2017-01-23 09:42:15.210789851 +0000
@@ -2333,6 +2333,10 @@
 \fBupdate-smart-drivedb\fP(8).
 .\" %ENDIF ENABLE_UPDATE_SMART_DRIVEDB
 
+.SH WARNING
+The smartctl utility accesses the internals of your system and its
+incorrect usage may render your system inoperable.
+
 .SH REFERENCES
 Please see the following web site for more info:
 \fBhttp://www.smartmontools.org/\fP
